packages/suite/src/views/settings/SettingsGeneral/Experimental.tsx

```diff
@@ -0,0 +1,109 @@
+import styled from 'styled-components';
+import { AnimatePresence, motion } from 'framer-motion';
+import { Checkbox, Switch } from '@trezor/components';
+import { spacingsPx } from '@trezor/theme';
+
+import { SUITE } from 'src/actions/suite/constants';
+import { ActionColumn, SectionItem, TextColumn, Translation } from 'src/components/suite';
+import {
+    ExperimentalFeature,
+    ExperimentalFeatureTitle,
+    ExperimentalFeatureDescription,
+} from 'src/constants/suite/experimental';
+import { useDispatch, useSelector } from 'src/hooks/suite';
+
+const FeatureLineWrapper = styled.div`
+    display: flex;
+    & + & {
+        padding-top: ${spacingsPx.md};
+        margin-top: ${spacingsPx.md};
+        border-top: 1px solid ${({ theme }) => theme.borderElevation2};
+    }
+`;
+
+const FeatureLine = ({
+    feature,
+    features,
+}: {
+    feature: ExperimentalFeature;
+    features: ExperimentalFeature[];
+}) => {
+    const dispatch = useDispatch();
+    const checked = features.includes(feature);
+    const titleId = ExperimentalFeatureTitle[feature];
+    const descId = ExperimentalFeatureDescription[feature];
+
+    return (
+        <FeatureLineWrapper>
+            <TextColumn
+                title={titleId ? <Translation id={titleId} /> : feature}
+                description={descId && <Translation id={descId} />}
+            />
+            <ActionColumn>
+                <Checkbox
+                    isChecked={checked}
+                    onClick={() =>
```

Comment:
nit: I would prefer when its multiline to extraxct it to funcion

```ts
const handleOnClick = () => { 
...
```

Reply:
Less lines => why not.